src/test/java/sorinfoca/tests/MyTests.java

```diff
@@ -0,0 +1,100 @@
+package sorinfoca.Tests;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import sorinfoca.driver.BrowserManager;
+
+import static org.testng.AssertJUnit.assertEquals;
+import static org.testng.AssertJUnit.assertTrue;
+
+public class MyTests extends TestConfig {
+
+    WebDriver driver;
+
+    @DataProvider(name = "LoginDataProvider")
```

Comment:
Only as a suggestion, consider to put the name attribute for the @DataProvider the same as the method name. For example, `@DataProvider(name = "loginDataProvider")`
`public static Object[][] loginDataProvider() {`
In case you decide to follow this suggestion, please do the same for the other @DataProvider too. Thanks!